packages/lib/versionInfo.ts

```diff
@@ -32,9 +69,11 @@ export default function versionInfo(packageInfo: any) {
 		console.info(gitInfo);
 	}
 
+	const pluginList: PluginList = getPluginLists();
```

Comment:
Type is inferred here, so to avoid cluttering the code you don't need to write it down. In general we prefer to rely on infer types (maybe that's something we should add to the eslint rules).

Reply:
Fixed.